packages/frontend/src/components/SettingsScheduledDeliveries/index.tsx

```diff
@@ -21,6 +27,58 @@ interface ProjectUserAccessProps {
     projectUuid: string;
 }
 
+const getContent = (scheduler: Scheduler, projectUuid: string) => {
+    return scheduler.dashboardUuid !== null ? (
+        <Anchor
+            href={`/projects/${projectUuid}/dashboards/${scheduler?.dashboardUuid}/view`}
+            target="_blank"
+        >
+            View dashboard
+        </Anchor>
+    ) : (
+        <Anchor
+            href={`/projects/${projectUuid}/saved/${scheduler?.savedChartUuid}/view`}
+            target="_blank"
+        >
+            View chart
+        </Anchor>
+    );
+};
+
+const getDetails = (scheduler: SchedulerAndTargets, user?: User) => {
```

Comment:
Since we are returning a component, these functions could be functional components?

Reply:
Yep, done